code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -92,3 +92,96 @@
 		to_chat(M, span_warning("You don't want to hurt anyone, just give them hugs!"))
 		M.a_intent = INTENT_HELP
 	.= FALSE
+
+/obj/item/clothing/gloves/bracer/cuffs
+	name = "rabid cuffs"
+	desc = "Chainless manacles fashioned after the hungriest of slaughter demons. Grants the wearer a similar hunger that can be sated in a similar way."
+	icon_state = "cuff"
+	item_state = "cuff"
+	var/obj/effect/proc_holder/swipe/swipe_ability
+
+/obj/item/clothing/gloves/bracer/cuffs/Initialize()
+	. = ..()
+	swipe_ability = new(swipe_ability)
+
+/obj/item/clothing/gloves/bracer/cuffs/equipped(mob/living/user, slot)
+	. = ..()
+	if(ishuman(user) && slot == ITEM_SLOT_GLOVES)
+		user.AddAbility(swipe_ability)
+
+/obj/item/clothing/gloves/bracer/cuffs/dropped(mob/living/user)
+	. = ..()
+	user.RemoveAbility(swipe_ability)
+
+obj/effect/proc_holder/swipe
+	name = "swipe"
+	desc = "Swipe at a target area, dealing damage and consuming dead creatures to heal yourself. People are ineligible for total consumption. Creatures take 30 damage and heal the most while people and cyborgs take 10 damage and heal for the least. People that have been thoroughly burned and bruised heal you for a bit more than those that aren't."
```

Comment:
```suggestion
	desc = "Swipe at a target area, dealing damage and consuming dead entities to heal yourself. People are ineligible for total consumption. Creatures take 30 damage, and give the most healing, while people and cyborgs take 10 damage and heal for the least. People who have been thoroughly burned and bruised heal you for slightly more!"
```

Reply:
Is the damage intended to be applied to corpses? Or is this on living targets? It's not entirely clear, as you talk about swiping and then consuming dead things in the start. May want to reorder the tooltip into a section about damage, and interactions with living entities, then corpse interactions and healing.

Reply:
the damage is intended for whoever the attack hits regardless of living or dead status, the conuming thing is just for dead targets; will get to work on making the description clearer though